openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -201,9 +201,9 @@ def test_basic_auth(self):
         # ensure that having basic auth headers in the mix does not break anything
         self.assertHttpOK(
             self.request_with_auth("get", self.LIST_URI,
-                                   **self.basic_auth("someuser", "somepass")))
+                                   **self.basic_auth(b"someuser", b"somepass")))
```

Comment:
See [Design Principle 1](https://openedx.atlassian.net/wiki/spaces/AT/pages/998998951/Python+3+Conversion).  Can we keep strings as strings as they are passed through the codebase - and then convert them to bytes only at the periphery?

Reply:
yes, you are correct. I changed my modifications to better fit Design principle 1.